db/migrate/20180703164027_change_status_of_comments.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeStatusOfComments < ActiveRecord::Migration[5.1]
+  def up
+    Comment.all.each do |comment|
+      comment.status = 1
+      comment.save({})
```

Comment:
is the `{}` required? Were you able to test this out locally? 

In general it looks good to me!

Reply:
yes, migration worked locally.

Reply:
@Gauravano i'm curious, what's the purpose of this `comment.status = 1`

Reply:
Hi @siaw23, actually to make spam moderation system for comments, comments would come with 2 status values `status = 1` for valid and `status = 0` for spam.